action.yml

```diff
@@ -28,6 +28,14 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - name: List installed packages
+      shell: bash
+      run: apt list --installed > installed_packages.txt
+    - name: Upload installed packages list
+      uses: actions/upload-artifact@v2
+      with:
+        name: installed-packages
+        path: installed_packages.txt
```

Comment:
Don't we want to print it to std?
So that we would be able to see it in the GH Action logs of other tenants?